src/utils/index.ts

```diff
@@ -1,3 +1,6 @@
-export function areAddressesEqual(address1: string, address2: string): boolean {
-  return address1.toLowerCase() === address2.toLowerCase()
+export const sameString = (str1?: string, str2?: string): boolean => {
+  if (!str1 || !str2) {
```

Comment:
but if both are `undefined` then they are the same :D

Reply:
but why would it accept undefined as an argument? that doesn't make sense. the parameters have to be a string in order for the function to work properly, if one of the arguments you want to pass can be undefined, use an if statement and don't call the function

Reply:
even if you don't accept `undefined`, same applies to the empty string ... if both are empty this should return `true` but as `!""` evaluates to `true` this function will return `false` if both strings are empty ... which is wrong according to the name.

Reply:
If you don't accept undefined you don't need this check